WcaOnRails/app/views/static_pages/wca_workbook_assistant.html.erb

```diff
@@ -11,62 +11,62 @@
 <div class="container">
   <h1><%= yield(:title) %></h1>
   <div class="text-center">
-    <a href="/files/wca-workbook-assistant-2.3.jar">Download WCA Workbook Assistant v2.3</a><br/><br/>
-    <span>Last update: April 7, 2015</span><br/>
-    (<%= link_to "version history", wca_workbook_assistant_versions_path %>)
+    <a href="/files/wca-workbook-assistant-2.3.jar"><%= t('workbook_assistant.download') %></a><br/><br/>
```

Comment:
Let's leave the version number here, otherwise we will need to change all translations whenever it changes.